catalog/bucket/bucket.yaml

```diff
@@ -14,14 +14,13 @@
 apiVersion: storage.cnrm.cloud.google.com/v1beta1
 kind: StorageBucket
 metadata:
-  name: blueprints-project-bucket # kpt-set: ${project-id}-${name}
-  namespace: config-control
+  name: bucket
   annotations:
     cnrm.cloud.google.com/force-destroy: "false"
-    cnrm.cloud.google.com/project-id: blueprints-project # kpt-set: ${project-id}
+    cnrm.cloud.google.com/project-id: PLACEHOLDER
```

Comment:
Even for this, I think `sample-project-id` would be a better placeholder. For one thing, it's a reminder that project IDs are hyphen-delineated and always lowercase.

Reply:
Thank you for the explanation. We try to use the same string if possible for all required configurable fields, so it's easy to build the convention and searc+replace. Do you have suggestions what could be a good value? how about "sample"?

Reply:
For convention and searching, I think it would be sufficient to search for all values with "example" or "sample." (Ex. `sample-project-id`) We can choose one as the convention and document it.

Reply:
Thank you. `sample` works for me, one less character.